src/components/claims/claims-list/BackendServedClaimsList.tsx

```diff
@@ -0,0 +1,130 @@
+import { LinkRow } from 'components/shared'
+import * as moment from 'moment'
+import * as React from 'react'
+import { Table } from 'semantic-ui-react'
+import { history } from 'store'
+import {
+  Claim,
+  ClaimSearchFilter,
+  ClaimSortColumn,
+  ClaimsStore,
+} from '../../../store/types/claimsTypes'
+import BackendPaginatorList from '../../shared/paginator-list/BackendPaginatorList'
+
+export interface BackendServedClaimsListProps {
+  claims: ClaimsStore
+  claimsRequest: (filter: ClaimSearchFilter) => void
+}
+
+export default class BackendServedClaimsList extends React.Component<
```

Comment:
We prefer using Stateless Functional Components:

```suggestion
export const BackendServedClaimsList: React.SFC<
```

Reply:
For the handler functions you may define them inline, we've decided that this is acceptable code style for our projects.

The sub-components can be defined as their own Stateless Functional Components 🙂

Reply:
Fixed it